### PR TITLE
Update version to 12.5.0.alpha.0 for gem release.

### PR DESCRIPTION
**Pull request by mcquin**
Releasing a 12.5.0.alpha.0 gem which includes changes to chef-config which are needed by ohai.

@chef/client-core

### CODE REVIEW

**Comment by danielsdeleo** [comment]
I think this is good to :ship: but remember not to merge :)

File: chef-config/lib/chef-config/version.rb [hunk 1]
```diff
@@ -20,6 +20,6 @@
 #!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 module ChefConfig
-  VERSION = '12.5.0.current.0'
+  VERSION = '12.5.0.alpha.0'
```
Comment (tyler-ball) [inline chef-config/lib/chef-config/version.rb]: Until we pull `ChefConfig` into its own gem, why not set `VERSION = Chef::VERSION`

**Comment by tyler-ball** [comment]
:+1:

File: chef-config/lib/chef-config/version.rb (hunk 1 above)
Comment (mcquin) [inline chef-config/lib/chef-config/version.rb]: Wouldn't chef-config have to depend on chef in order to do that?

Comment (danielsdeleo) [inline chef-config/lib/chef-config/version.rb]: Yeah, Claire's reasoning is why we don't do that. `chef-config` is its own gem, it's just in the same repo as chef (the same way rails is). So we have a `VERSION` file at the root and a rake task to do the update everywhere.

Comment (tyler-ball) [inline chef-config/lib/chef-config/version.rb]: Thanks for the explanation!  Carry on

**Pull request closed without merging**